File: src/main/java/me/nobaboy/nobaaddons/keybinds/CommandKeyBind.java
package me.nobaboy.nobaaddons.keybinds;

import me.nobaboy.nobaaddons.util.ChatUtils;
import org.lwjgl.input.Keyboard;

public class CommandKeyBind extends NobaKeyBind {
	public CommandKeyBind(String name, int key, String command) {
		super(name, key, () -> ChatUtils.sendCommand(command));
	}

	public CommandKeyBind(String name, String command) {
		this(name, Keyboard.KEY_NONE, command);
	}
}

File: src/main/kotlin/me/nobaboy/nobaaddons/keybinds/CommandKeyBind.kt
package me.nobaboy.nobaaddons.keybinds

import me.nobaboy.nobaaddons.util.ChatUtils
import org.lwjgl.input.Keyboard

class CommandKeyBind(name: String, key: Int = Keyboard.KEY_NONE, command: String) : NobaKeyBind(
    name, key, { ChatUtils.queueCommand(command) }
) {
    constructor(name: String, command: String) : this(name, Keyboard.KEY_NONE, command)
}